Bottom: bb4a887b294ded479b4881ae40937b4f778fcb32
Top:    26e493c01e454014a7d5441c87750df631734c4f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 16:35:11 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index 49f011f..1c1a504 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
